automathor.py

```
@@ -0,0 +1,187 @@
+
+
+logging.basicConfig(level=logging.INFO)
+
+automathor = Automathor()
+automathor.run_git_grep()
+automathor.process_matches()
+i = 0
+metadata = {}
+os.makedirs("tmp", exist_ok=True)
+for match in automathor.matches:
+    i += 1
```

Ruff (0.8.2)

Use `enumerate()` for index variable `i` in `for` loop

(SIM113)